src/client.rs

```diff
@@ -3,76 +3,117 @@
 // Licensed under the MIT License, <LICENSE or http://opensource.org/licenses/MIT>.
 // This file may not be copied, modified, or distributed except according to those terms.
 
-use Packet;
-use futures::{Async, BoxFuture};
-use futures::stream::Empty;
+use WireError;
+use bincode::serde::DeserializeError;
+use framed::Framed;
+use futures::{self, Async, Future};
+use serde::{Deserialize, Serialize};
 use std::fmt;
 use std::io;
-use tokio_proto::pipeline;
+use tokio_core::net::TcpStream;
+use tokio_core::reactor;
+use tokio_proto::easy::{EasyClient, EasyResponse, multiplex};
 use tokio_service::Service;
-use util::Never;
 
 /// A client `Service` that writes and reads bytes.
```

Comment:
I think EasyClient is probably a wrapper around a tokio service. `EasyClient` is an implementation detail though, so we probably don't need to mention that. I think the important thing is that `Client` impls `tokio_service::Service`. I clarified a bit.